nanomq/sub_handler.c

```diff
@@ -84,6 +84,10 @@ decode_sub_message(nano_work *work)
 					        &len_of_str);
 					sub_pkt->user_property.strpair
 					    .len_key = len_of_str;
+					if (len_of_topic < 0) {
```

Comment:
Actually, I donot find len_of_topic be used before, so why you check it here?

Reply:
should be len_of_str. @wanghaEMQ  fix your careless mistake